volatility3/cli/__init__.py

```diff
@@ -313,6 +322,8 @@ def run(self):
 
         if partial_args.offline:
             constants.OFFLINE = partial_args.offline
+        elif partial_args.remote_isf_url:
+            constants.REMOTE_ISF_URL = partial_args.remote_isf_url
```

Comment:
I'm just wondering whether we should include (for all cases, not just when the parameter's been specified) a `vollog.info()` call that records what remote ISF server may have been used?  I can't figure out if it would be useful in helping fix problems or not?  I'm thinking probably not, because after it's been used, the file's cached and its link to a remote server goes away, so I think it's fine as is.